### ISSUE TITLE
Text selection results in gray overlay

### ISSUE THREAD

**Issue by razh** [wojtekmaj/react-pdf#1925]
### Before you start - checklist

- [X] I followed instructions in documentation written for my React-PDF version
- [X] I have checked if this bug is not already reported
- [X] I have checked if an issue is not listed in [Known issues](https://github.com/wojtekmaj/react-pdf/wiki/Known-issues)
- [X] If I have a problem with PDF rendering, I checked if my PDF renders properly in [PDF.js demo](https://mozilla.github.io/pdf.js/web/viewer.html)

### Description

The CSS added in #1880 results in a gray overlay whenever text is selected:

```diff
diff --git a/packages/react-pdf/src/Page/TextLayer.css b/packages/react-pdf/src/Page/TextLayer.css
index 727c21127..094eff504 100644
--- a/packages/react-pdf/src/Page/TextLayer.css
+++ b/packages/react-pdf/src/Page/TextLayer.css
@@ -105,8 +105,9 @@
   user-select: none;
 }
 
-.textLayer .endOfContent.active {
+.textLayer.selecting .endOfContent {
   top: 0;
+  background: gray;
 }
 
 .hiddenCanvasElement {
```

https://github.com/wojtekmaj/react-pdf/pull/1880/files#diff-44343a2a08329e0e1279454e436a982253618ee02b740f26dfe07e21c3c7b3db

### Steps to reproduce

1. Run `yarn dev`
2. Select a PDF.
3. Attempt to select text.

Before selecting text|After selecting text
:-|:-
![CleanShot 2024-12-19 at 16 15 28@2x](https://github.com/user-attachments/assets/18c59a80-9f83-4068-9ed7-2f5bfce1ce1d)|![CleanShot 2024-12-19 at 16 15 42@2x](https://github.com/user-attachments/assets/dde79850-c38d-4c3d-a3a9-9718389ade61)

### Expected behavior

No gray overlay.

### Actual behavior

`.endOfContent` appears to cover the entire PDF with a gray background.

### Environment

- **Browser (if applicable)**:
- **React-PDF version**: 9.2.0
- **React version**: 
- **Bundler name and version (if applicable)**:

**Comment by landegt** [comment]
Can confirm this issue with react-pdf@9.2.0. The .endOfContent div expands to cover the entire document with a gray background during text selection.

**Comment by wojtekmaj** [comment]
Thanks, fixed in 9.2.1!
